src/utils/event.ts

```diff
@@ -85,3 +86,7 @@ export const getEventStatus = (event: VenueEvent) => {
     return `Starts ${formatUtcSecondsRelativeToNow(event.start_utc_seconds)}`;
   }
 };
+
+export const sortEventsByStartUtcSeconds = (
+  events: WithVenueId<WithId<VenueEvent>>[]
+) => events.sort((a, b) => a.start_utc_seconds - b.start_utc_seconds);
```

Comment:
Similar to `chatSort` in `src/utils/chat.ts`, we should just be making the sorting function here, not the whole `.sort` part of it, which is redundant. eg.

```suggestion
export const eventSort = (a: VenueEvent, b: VenueEvent) => a.start_utc_seconds - b.start_utc_seconds
```

So that it can then be used in any `.sort` as:

```typescript
fooEvents.sort(eventSort)
```

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1538/commits/6e240c48936c15f5ee1103f2ac1e7fb4f6f1ec67